src/test/java/co/rsk/federate/signing/hsm/message/ReleaseCreationInformationGetterTest.java

```diff
@@ -323,29 +427,214 @@ void getTxInfoToSign_whenTransactionReceiptNotFoundInSubsequentBlock_shouldThrow
 
         Block secondBlock = mock(Block.class);
         when(secondBlock.getHash()).thenReturn(secondBlockHash);
-        when(secondBlock.getTransactionsList()).thenReturn(Collections.singletonList(transactionInSecondBlock));
+        when(secondBlock.getTransactionsList()).thenReturn(
+            Collections.singletonList(transactionInSecondBlock));
 
         BlockStore blockStore = mock(BlockStore.class);
         when(blockStore.getBlockByHash(blockHash.getBytes())).thenReturn(block);
         when(blockStore.getChainBlockByNumber(anyLong())).thenReturn(secondBlock);
 
         ReceiptStore receiptStore = mock(ReceiptStore.class);
-        when(receiptStore.getInMainChain(rskTxHash.getBytes(), blockStore)).thenReturn(Optional.of(transactionInfo));
-        when(receiptStore.getInMainChain(rskTxHashInSecondBlock.getBytes(), blockStore)).thenReturn(Optional.empty());
+        when(receiptStore.getInMainChain(rskTxHash.getBytes(), blockStore)).thenReturn(
+            Optional.of(transactionInfo));
+        when(receiptStore.getInMainChain(rskTxHashInSecondBlock.getBytes(), blockStore)).thenReturn(
+            Optional.empty());
 
         ReleaseCreationInformationGetter pegoutCreationInformation = new ReleaseCreationInformationGetter(
-                receiptStore,
-                blockStore
+            receiptStore,
+            blockStore
         );
 
         BtcTransaction pegoutBtcTransaction = mock(BtcTransaction.class);
         byte[] btcTxHash = TestUtils.createHash(2).getBytes();
         when(pegoutBtcTransaction.getHash()).thenReturn(Sha256Hash.wrap(btcTxHash));
 
-        assertThrows(HSMReleaseCreationInformationException.class, () -> pegoutCreationInformation.getTxInfoToSign(
-            2,
-            rskTxHash,
-            pegoutBtcTransaction
-        ));
+        assertThrows(HSMReleaseCreationInformationException.class,
+            () -> pegoutCreationInformation.getTxInfoToSign(
+                2,
+                rskTxHash,
+                pegoutBtcTransaction
+            ));
+    }
+
+    @ParameterizedTest
+    @MethodSource("getTxInfoToSignArgProvider")
+    void getTxInfoToSign_whenBatchPegoutHasPegoutTransactionCreatedEvent_returnsOk(
+        int version,
+        byte[] serializedOutpointValues,
+        List<Coin> expectedOutpointValues
+    ) throws HSMReleaseCreationInformationException {
+        // Arrange
+        List<LogInfo> logs = new ArrayList<>();
+
+        ECKey senderKey = new ECKey();
+        RskAddress senderAddress = new RskAddress(senderKey.getAddress());
+        LogInfo updateCollectionsLog = createUpdateCollectionsLog(senderAddress);
+        logs.add(updateCollectionsLog);
+
+        Coin pegoutAmount = pegoutBtcTx.getInputSum();
```

Comment:
Not so sure about this. The sum of all the inputs does not equal the pegout amount. There will be change

Reply:
You are right. I will set it equal to the minimum `bridgeConstants.getMinimumPegoutTxValue()`. A batch pegout containing a single pegout request sends the minimum. Wdyt?

Reply:
Done